pkg/services/star/api/api.go

```diff
@@ -67,20 +62,8 @@ func (api *API) GetStars(c *contextmodel.ReqContext) response.Response {
 	}
 
 	uids := []string{}
-	if len(iuserstars.UserStars) > 0 {
-		var uids []string
```

Comment:
^^^ or we could just remove this line

the local uids variable means we always return an empty list